packages/redux/src/contents/utils.ts

```diff
@@ -162,7 +196,7 @@ export const getBestRankedCommercePageUsingDefaultStrategy = (
 
 /**
  * Method to return the best ranked commerce page with all components from other
- * pages without duplicates.
+ * pages without duplicates. This implementation only works with content from new Content Tool
```

Comment:
Question: Will this be a breaking change for current consumers of this function?

Reply:
This implementation is not used by any tenant. Currently, the only tenant that uses the merge strategy is Harrods, and they have their own implementation in their bff. This is a fix to BG implement merge strategy using blackout now